.github/workflows/image.yaml

```diff
@@ -43,6 +59,14 @@ jobs:
       - name: Login to Quay Registry
         if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
         run: echo ${{ secrets.QUAY_PASSWORD }} | docker login -u ${{ secrets.QUAY_USERNAME }} --password-stdin quay.io
+      - if: ${{ steps.cache-npm.outputs.cache-hit != 'true' }}
+        name: List the state of node modules
+        continue-on-error: true
+        working-directory: ./internal/webui/public
+        run: npm list
+      - name: Install WebUI dependencies
+        working-directory: ./internal/webui/public
```

Comment:
I think it's best to fit those in the Earthfile target rather in the github action, so when someone calls "build" it builds the binary, with deps included